src/ebmc/bdd_engine.cpp

```diff
@@ -131,7 +131,7 @@ int bdd_enginet::operator()()
 {
   try
   {
-    int result=get_model();
+    int result = get_transition_system();
     if(result!=-1) return result;
```

Comment:
Not for this PR, but still: the `int` return type should be replaced by an enum. This is rather difficult to understand, and also seems hard to maintain.